src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: { name, age, sex, isMarried, partnerName },
+}) => {
+  const partnerSex = sex === 'f' ? 'husband' : 'wife';
```

Comment:
The logic for determining `partnerSex` assumes traditional gender roles, which may not be accurate. Consider using the `partnerName` and `isMarried` properties directly without assuming the partner's gender based on the person's sex.